awscrt/s3.py

```diff
@@ -401,6 +418,13 @@ def make_request(
                         this is the final response's status code. If the operation
                         failed for another reason, None is returned.
 
+                    *   `did_validate_checksum` (Optional[bool]):
```

Comment:
it's not optional, it's always true or false
```suggestion
                    *   `did_validate_checksum` (bool):
```